src/applications/vaos/appointment-list/components/RequestedAppointmentDetailsPage.jsx

```diff
@@ -73,6 +77,11 @@ function RequestedAppointmentDetailsPage({
   const typeOfCareText = lowerCase(appointment?.type?.coding?.[0]?.display);
   const facilityId = getVAAppointmentLocationId(appointment);
   const facility = facilityData?.[facilityId];
+  const isCCRequest =
+    appointment.vaos.appointmentType === APPOINTMENT_TYPES.ccRequest;
+  const practitionerName = appointment.contained
+    .filter(res => res.resourceType === 'Practitioner')
+    .map(provider => provider.practitionerRole?.[0].location?.[0].display);
```

Comment:
I think there is a helper called `getPractitionerLocationDisplay()` from the appointment service you can use to get this practitioner name.  If not using that, you probably want to put an an additional optional after the array indexes:

`provider.practitionerRole?.[0]?.location?.[0]?.display`